### DISCUSSION TITLE
Any known security vulnerabilities in Airflow version 2.1.0

### DISCUSSION THREAD

**Discussion by VenkiAnu** [apache/airflow#17510] [Q&A]
Hello,

We just installed version 2.1.0 last month. Are there any known security vulnerabilities? Specifically around the below:
•	Cross-site Scripting 
•	Privilege Escalation 
•	Improper Authentication 
Any insights appreciated.

**Comment by potiuk** [comment]
All the security issues about Apache Airflow are handled via https://www.apache.org/security/committers.html process and end up as CVEs in the public security databases such as mitre. Depending on what your security team uses, you might use different sources to feed them as those issues are public and 'standardised'

If there are any unpublished security issues they are (following the process above) discussed between that researchers, Apache Security and PMC in private list and responsibly disclosed when they are fixed. Until this is done we cannot share any of those (even if such CVEs are already known to us). This is extremely important for the 'responsible disclosure' part.

Publicly asking here and expecting to see anything more than this publicly announced ones is therefore an ask to disclose any of those issues in 'irresponsible' way. So do not expect anything more than the publicly disclosed ones (until it is safe to disclose them and this is decided by the PMC). You can subscribe to 'announce' list (see the ASF process) to get notified when fixed issues are disclosed.
